package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "node_mentoring",
+  "version": "1.0.0",
+  "description": "Node.js mentoring",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js",
+    "task1": "nodemon src/task1.js",
```

Comment:
you should use `babel-node` to run your code. Babel node will transpile your es6 modules to common js modules, so node (version 12 and below) will be able to run it